ci-operator/config/openshift/azure-storage-azcopy/openshift-azure-storage-azcopy-main.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+  - dockerfile_path: Dockerfile
+    inputs:
+      ocp_4.16_base:
+        as:
+          - registry.ci.openshift.org/ocp/4.16:base
+    to: azure-storage-azcopy-base
+promotion:
```

Comment:
`promition`: opt-in for automatic branching: https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching